tools/Propulsion.Tool/Args.fs

```diff
@@ -253,3 +260,36 @@ module Dynamo =
         member x.CreateCheckpointStore(group, cache, storeLog) =
             let context = DynamoStoreContext.create indexReadClient.Value
             Propulsion.Feed.ReaderCheckpoint.DynamoStore.create storeLog (group, checkpointInterval) (context, cache)
+
+module Mdb =
+    open Configuration.Mdb
+    open Npgsql
+    type [<NoEquality; NoComparison>] Parameters =
+        | [<AltCommandLine "-c">]           ConnectionString of string
+        | [<AltCommandLine "-cp">]          CheckpointConnectionString of string
+        | [<AltCommandLine "-s">]           Schema of string
+        | [<AltCommandLine "-cat">]         Tranches of Propulsion.Feed.TrancheId
```

Comment:
```suggestion
        | [<AltCommandLine "-cat">]         Category of string
```

Reply:
Argh, yeah, missed a spot.

Reply:
Shouldn't it be `Categories` as you can give it multiple ones?

Reply:
Nope, absolutely should be `Category`, done!